source/server/getting-started/implementations/spongeforge.rst

```diff
@@ -92,7 +92,7 @@ Dedicated Servers
 Installing Forge via Commandline
 --------------------------------
 
-1. Visit the `Minecraft Forge website <https://files.minecraftforge.net/>`_ and click "Show all downloads" to view the full
+1. Visit the `Minecraft Forge website <http://files.minecraftforge.net/>`_ and click "Show all downloads" to view the full
```

Comment:
Try using two underscores here (and below). It's a strange warning and I've no idea why it only happens with https, but that seems to fix it according to https://github.com/sphinx-doc/sphinx/issues/3921